src/controllers/users.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const userService = require('../services/users');
+
+const getAll = (req, res) => {
+  const users = userService.getAll();
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { userId } = req.params;
+  const foundedUser = userService.getById(userId);
+
+  if (!foundedUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundedUser);
+};
+
+const add = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = userService.create(name);
+
+  res.statusCode = 201;
+  res.send(newUser);
+};
+
+const remove = (req, res) => {
+  const { userId } = req.params;
+  const foundedUser = userService.getById(userId);
```

Comment:
The userService.getById() function is called without await, which can lead to an unhandled promise rejection if the promise is rejected. You should use async/await or promise chaining to handle the promise properly.